.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: "\U0001F41BBug report"
+description: Create a report to help us improve Optuna.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please write a clear and concise description of what the bug is.
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Please write a clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please give us some environment information. You can get this information by typing the following.
+        ```
+        python -c 'import optuna; print(optuna.__version__)'
+        python -c 'import platform; print(platform.python_version())'
+        python -c 'import platform; print(platform.platform())'
+        ```
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Error messages, stack traces, or logs
+      description: Please copy and paste any relevant error messages, stack traces, or log output. 
+      render: shell
+    validations:
+       required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Please provide how we reproduce your reported bugs. If possible, it is highly recommended to provide the reproducible example codes.
+      placeholder: |
+        1.
+        2.
+        3.
+        ```python
+        # python code
+        ```
```

Comment:
It feels to me a bit unfriendly because the placeholder disappears when a character is put on the form. How about making it a code block as in environment section?

Reply:
Sounds reasonable. I'll fix it.

Reply:
I use `value` instead of `placeholder`. Then, the code block is a default value and does not disappear when a character is put on the form. PTAL.